tests/test_se3.py

```diff
@@ -159,3 +160,45 @@ def test_SE3_chaining_transforms() -> None:
     combined_R = get_yaw_angle_rotmat(theta0 + theta1)
     assert np.allclose(fr2_se3_fr0.rotation, combined_R)
     assert np.allclose(fr2_se3_fr0.translation, np.zeros(3))
+
+
+def test_SE3_transform_point_cloud_optimized() -> None:
+    """Ensure that our transform_point_cloud() implementation is faster than a naive implementation.
+
+    We use 1k trials, with a point cloud of 100k points. The chosen SE(3) transformation is arbitrary.
+    """
+
+    def transform_point_cloud_slow(point_cloud: np.ndarray, transform_matrix: np.ndarray) -> np.ndarray:
+        """Slow, unoptimized method -- allocated unnecessary array for homogeneous coordinates."""
+        # convert to homogeneous
+        num_pts = point_cloud.shape[0]
+        homogeneous_pts = np.hstack([point_cloud, np.ones((num_pts, 1))])
+        transformed_point_cloud = homogeneous_pts.dot(transform_matrix.T)
+        return transformed_point_cloud[:, :3]
+
+    num_trials = 1000
+    num_pts = 100000
```

Comment:
Can we move this up as module level constants?

Reply:
not sure if they apply to the module, since they apply to just a single test out of 8. I'm worried that it might confuse people, since they might assume it applies to each unit test